php/Dockerfile

```diff
@@ -4,5 +4,4 @@ RUN DEBIAN_FRONTEND=noninteractive apt install -y \
       libtool-bin \
       bison \
       re2c \
-      libsqlite3-dev \
```

Comment:
Just a reminder that you will need to rebuild and re-publish the php-builder image once this is merged.

Reply:
Published. Tested